_i18n/en.yml

```diff
@@ -501,6 +501,15 @@ faq:
   avuln: >
     The Monero community has created a series of videos called "Breaking Monero", where potential Monero vulnerabilities are explored and discussed. There are 14 videos, with each exploring a different subject. Check out <a href="https://www.monerooutreach.org/breaking-monero/">the videos on monerooutreach.org</a>.
   vulnspotify: Available on Spotify as podcast
+  qcriminals: Is Monero for criminals?
```

Comment:
```suggestion
  qcriminals: Was Monero created for criminals?
```

Reply:
Here it's more like: "Monero (like any other currency) is for everyone to use. It was not created specifically for criminals (like any other currency).

Reply:
That's not my point. My point is "Is Monero now the currency for criminals?". The point is not what it was created for, but how it's used.